### PR TITLE
Basic structure of GraphQL proxy

### PR DESCRIPTION
**Pull request by jorilindell**

### CODE REVIEW

File: graphql-proxy/package.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react": "^7.16.0",
+    "eslint-plugin-react-hooks": "^1.7",
+    "eslint-plugin-simple-import-sort": "^4.0.0",
+    "prettier": "^1.18.2"
+  },
+  "jest": {
+    "setupFilesAfterEnv": [
+      "./test/jest.setup.js"
```
Comment (jimppa) [inline graphql-proxy/package.json]: This file does not exist, causing `yarn test` to fail.

**Comment by badrange** [comment]
Looks good to me; and it would be great with instructions for debugging (even configurations in .vscode/launch.json committed to repo) here too.

File: graphql-proxy/src/schema/resolvers.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { merge } from "lodash";
```
Comment (jimppa) [inline graphql-proxy/src/schema/resolvers.ts]: I think this imports all of lodash, import instead just the merge function from `lodash/merge`

File: graphql-proxy/package.json (hunk 1 above)
Comment (jorilindell) [reply graphql-proxy/package.json]: FIxed, setup file was on wrong folder

File: graphql-proxy/src/schema/resolvers.ts (hunk 2 above)
Comment (jorilindell) [reply graphql-proxy/src/schema/resolvers.ts]: Changed

File: graphql-proxy/src/schema/schema.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import dummy from "./dummy/typeDefs";
```
Comment (jimppa) [inline graphql-proxy/src/schema/schema.ts]: Does this file have any purpose? It's identical to `typeDefs.ts`, but this file is not used.

**Pull request merged**